pkg/proc/variables.go

```diff
@@ -2278,7 +2283,9 @@ func (cm constantsMap) Get(typ godwarf.Type) *constantType {
 	typepkg := packageName(typ.String()) + "."
 	if !ctyp.initialized {
 		ctyp.initialized = true
-		sort.Sort(constantValuesByValue(ctyp.values))
+		slices.SortFunc(ctyp.values, func(a, b constantValue) int {
```

Comment:
`return a.value - b.value`?

Reply:
We can't change to `return int(a.ID - b.ID)` because it will incorrectly sort `int64` values:

The [following program](https://go.dev/play/p/U2kpBXwBuPF) demonstrates that:
```go
package main

import (
	"cmp"
	"fmt"
	"math"
	"slices"
)

type constantValue struct {
	value int64
}

func main() {
	valuesSub := []constantValue{{math.MaxInt64}, {1}, {math.MinInt64}}
	valuesCmp := slices.Clone(valuesSub)
	slices.SortFunc(valuesSub, func(a, b constantValue) int { return int(a.value - b.value) })
	fmt.Println("Sorted valuesSub:", valuesSub) // Sorted valuesSub: [{1} {9223372036854775807} {-9223372036854775808}]

	slices.SortFunc(valuesCmp, func(a, b constantValue) int { return cmp.Compare(a.value, b.value) })
	fmt.Println("Sorted valuesCmp:", valuesCmp) // Sorted valuesCmp: [{-9223372036854775808} {1} {9223372036854775807}]

}
```